server/routes/creatingLicences/types/additionalConditionInputs/electronicMonitoringPeriod.ts

```diff
@@ -2,11 +2,19 @@ import { Expose, Type } from 'class-transformer'
 import { Validate } from 'class-validator'
 import SimpleDate from '../date'
 import ValidSimpleDate from '../../../../validators/simpleDateValidator'
+import DateIsBefore from '../../../../validators/dateIsBefore'
+import DateIsAfter from '../../../../validators/dateIsAfter'
 
 class ElectronicMonitoringPeriod {
```

Comment:
Doesnt the bug say we need this on AlcoholMonitoringPeriod aswell?

Reply:
It does mention 14d but gives no examples, so missed that bit. I'll check.

Reply:
Are you sure you looked closely? It does seem to be on that one too :-)